src/app/modules/main/communities/module.nim

```diff
@@ -95,7 +95,8 @@ type
 method setCommunityTags*(self: Module, communityTags: string)
 method setAllCommunities*(self: Module, communities: seq[CommunityDto])
 method setCuratedCommunities*(self: Module, curatedCommunities: seq[CommunityDto])
-proc buildTokensAndCollectiblesFromCommunities(self: Module)
+proc buildTokensAndCollectiblesFromAllCommunities(self: Module)
```

Comment:
Maybe it would be nice a comment here to better understand the difference between `Communities` and `AllCommunities`! ;)